flake.nix

```diff
@@ -77,6 +77,7 @@
 
     checks."x86_64-linux" = {
       default = self.apps."x86_64-linux".oom-test;
+      ref = pkgs.callPackage ./tests/ref.nix {};
```

Comment:
Doing tests through apps was admittedly not the most beautiful thing, I think my reason for doing it was to make it easy to find and run tests through the `nix-command` interface, and making them discoverable with `nix flake show` (which currently is broken, I have a fix for it in my non-pushed #99). 

I think we should do it in the same way thou, leaning very slightly towards a preference for doing it through the `apps`, but fine with doing it like this.

Reply:
It makes perfect sense to have just tests just in checks thou (since they're also discoverable in `nix flake show`), actually it's probably preferable, and the old way should be refactored.

Reply:
oom-tests pass on this, same with `nix build ./#checks.x86_64-linux.ref -L --rebuild --impure --option sandbox false`

:eyes: ... `--impure` ? That shouldn't be necessary?

`nix build .#checks.x86_64-linux.ref --option sandbox false --rebuild` should be sufficient, otherwise I made a mistake? :)

But, yeah.. Let me rewrite to callPackage as part of #102, will do, thanks!